bedrock/settings/base.py

```diff
@@ -24,29 +26,31 @@ def path(*args):
 
 
 # Is this a dev instance?
-DEV = False
+DEV = config('DEV', cast=bool, default=False)
 
-DEBUG = False
-TEMPLATE_DEBUG = DEBUG
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', cast=bool, default=False)
 
 # Production uses MySQL, but Sqlite should be sufficient for local development.
 # Our CI server tests against MySQL.
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'bedrock.db',
-    }
-}
-SLAVE_DATABASES = []
+DATABASES = config(
```

Comment:
In the normal case, the environment variable and .env file will only have a single DATABASE_URL, which should be fairly easy to read. I see the support for multiple DBs as a necessary interim hack that we can eventually get rid of entirely once the main servers are only talking to a read-only replica and we have separate instances entirely that can actually talk to the DB masters. However, even as an interim hack an .env file with both a RO and RW DB connection should be fairly readable:
```json
DATABASES='{"default": "mysql://rw_user:password@rw-prod-host/rw-db-name", "readonly": "mysql://ro_user:password@ro-prod-host/ro-db-name"}'
```